src/components/SearchSelectOrText.js

```diff
@@ -22,7 +22,7 @@ export const SearchSelectOrText = (props) => {
       if (item) {
         setCurrentDisplay(item.displayText);
       }
-    } else {
+    } else if (props.freetextValue) {
```

Comment:
Not sure if this changes the searchSelectOrText behavior too much, it seemed okay to me, but could use confirmation!